src/schemas/relatedWork.ts

```diff
@@ -2,159 +2,364 @@ import gql from 'graphql-tag';
+    LETTER
+    LIBGUIDES
     MODEL
-    OUTPUT_MANAGEMENT_PLAN
+    OTHER
+    PARATEXT
     PEER_REVIEW
     PHYSICAL_OBJECT
     PREPRINT
-    PROJECT
+    REFERENCE_ENTRY
     REPORT
+    RETRACTION
+    REVIEW
     SERVICE
     SOFTWARE
     SOUND
     STANDARD
-    STUDY_REGISTRATION
+    SUPPLEMENTARY_MATERIALS
     TEXT
     WORKFLOW
-    OTHER
+  }
+
+  "An award that funded a work"
+  type Award {
+    "The Award ID"
+    awardId: String
+  }
+
+  "An author of a work"
+  type Author {
+    "The author's ORCID ID"
+    orcid: String
+    "The author's first initial"
+    firstInitial: String
+    "The author's given name"
+    givenName: String
+    "The author's middle initials"
+    middleInitials: String
+    "The author's middle names"
+    middleNames: String
+    "The author's surname"
+    surname: String
+    "The author's full name"
+    full: String
+  }
+
+  "An institution of an author of a work"
+  type Institution {
+    "The name of the institution"
+    name: String
+    "The ROR ID of the institution"
+    ror: String
+  }
+
+  "A funder of a work"
+  type Funder {
+    "The name of the funder"
+    name: String
+    "The ROR ID of the funder"
+    ror: String
+  }
+
+  "The confidence of the related work match"
+  enum RelatedWorkConfidence {
+    "High confidence"
+    HIGH
+    "Medium confidence"
+    MEDIUM
+    "Low confidence"
+    LOW
+  }
+
+  "The status of the related work"
+  enum RelatedWorkStatus {
+    "The related work is pending assessment by a user"
+    PENDING
+    "The related work has been marked as related to a plan by a user"
+    ACCEPTED
+    "The related work has been marked as not related to a plan by a user"
+    REJECTED
+  }
+
+  type DoiMatch {
+    "Indicates whether the work's DOI was found on a funder award page associated with the plan"
+    found: Boolean!
+    "A confidence score representing the strength or reliability of the DOI match"
+    score: Float!
+    "The funder award entries and specific award pages where the DOI was found"
+    sources: [DoiMatchSource!]!
+  }
+
+  type DoiMatchSource {
+    "The parent award ID, if the award has a parent"
+    parentAwardId: String
+    "The award ID"
+    awardId: String!
+    "The award URL"
+    awardUrl: String!
+  }
+
+  type ContentMatch {
+    "The confidence score indicating how well the work content matches the plan content"
+    score: Float!
+    "Highlighted title showing relevant matched terms"
+    titleHighlight: String
+    "Highlighted fragments from the abstract showing relevant matched terms"
+    abstractHighlights: [String!]!
+  }
+
+  type ItemMatch {
+    "The position of the matched item within the work (zero-based index)"
+    index: Int!
+    "A confidence score representing how strongly this item matches the corresponding item in the plan"
+    score: Float!
+    "The specific fields that contributed to the match (e.g. name, orcid etc)"
+    fields: [String!]
+  }
+
+  extend type Mutation {
```

Comment:
Can you place these at the top? We've laid out the other files as queries, then mutations, then type and input declarations.